src/antlr/QueryStructureVisitor.ts

```diff
@@ -506,6 +506,7 @@ export abstract class QueryStructureVisitor<Result> extends AbstractParseTreeVis
     const relation = this.lastRelation;
     if (relation !== undefined) {
       const identifier = ctx.errorCapturingIdentifier().identifier();
+      // eslint-disable-next-line @typescript-eslint/no-unnecessary-condition
       const alias = identifier !== undefined ? common.stripQuote(identifier).name : relation.id;
```

Comment:
This one is interesting. I think the grammar doesn't allow CTEs with empty names, so `identifier` should always exist. WDYT @pavelgubin .

Reply:
Yes, quickly looked through, most of these cases make sense and need to be fixed. I will be able to address it after vacation on 13.09.

Reply:
@megaflo @pavelgubin 👍 Let's fix these in another PR.